src/data_hub/_api.py

```diff
@@ -48,7 +54,7 @@ def _get_client_methods():
     # the only reason we're creating this client is to
     # extract methods from it. So no need to
     # authenticate
-    client = DeeporiginData(token="", org_id="")
+    client = DeeporiginData(token="", org_id="").with_raw_response
```

Comment:
this change tells the client to not construct pydantic models and simply return the JSON the server gives us